src/index.js

```diff
@@ -43,16 +43,10 @@ const NOW_CONFIG_PATH = configFiles.getConfigFilePath();
 const NOW_AUTH_CONFIG_PATH = configFiles.getAuthConfigFilePath();
 
 const GLOBAL_COMMANDS = new Set(['help']);
-const insidePkg = process.pkg;
 
 epipebomb();
 
-// we only enable source maps while developing, since
-// they have a small performance hit. for this, we
-// look for `pkg`, which is only present in the final bin
-if (!insidePkg) {
-  sourceMap.install();
-}
+sourceMap.install();
```

Comment:
But now they are always installed instead of just during development, right?

Reply:
That was already the case once we stopped using `pkg`. Personally I'd prefer stack traces enabled for Sentry reports.

Reply:
I also can't think if a decent "is development" check anymore 🤔

Reply:
Agreed, maybe through `--debug` or an env var, but since it was the case already anyway I think it's good to go